tools/mo/openvino/tools/mo/main.py

```diff
@@ -580,14 +76,23 @@ def main(cli_parser: argparse.ArgumentParser, fem: FrontEndManager, framework: s
         log.error("---------------- END OF BUG REPORT --------------")
         log.error("-------------------------------------------------")
 
-    telemetry.send_event('mo', 'conversion_result', 'fail')
-    telemetry.end_session('mo')
-    telemetry.force_shutdown(1.0)
-    return 1
+    if ngraph_function is None:
+        return 1
+
+    output_dir = argv['output_dir'] if argv['output_dir'] != '.' else os.getcwd()
+    model_path = os.path.normpath(os.path.join(output_dir, argv['model_name'] + '.xml'))
+
+    # Ticket for fixing: 88606
+    print('[ WARNING ] MO Meta data is not serialized to IR.'.format(get_ir_version(argv)))
```

Comment:
Please use another snippet that was removed in this PR to still generate meta when it is called from within MO. We shouldn't stop generating meta from MO.

Reply:
Done.